verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -323,7 +381,9 @@ VName KytheFactsExtractor::PrintAnchorVName(const Anchor& anchor) {
 
 std::string KytheFactsExtractor::CreateScopeRelativeSignature(
     absl::string_view signature) const {
-  return absl::StrCat(signature, "#", vnames_context_.top().signature);
+  return vnames_context_.empty()
+             ? std::string(signature)
```

Comment:
Can you explain this change? Can't figure it out from the context.

Reply:
This is just to generalize the case for creating the scope-relative-signature, consider the below:

```
class bar;
endclass: bar

module foo();
   class zar;
   endclass;
endmodule
```

For the first class the vnames_context is empty but it's not for the second.

Reply:
That's very implicit and makes the code difficult to understand. Can you make this behavior more explicit? Even if it needs forking some code.